exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTxBase.java

```diff
@@ -0,0 +1,45 @@
+package com.exonum.binding.cryptocurrency.transactions;
+
+import static com.exonum.binding.cryptocurrency.CryptocurrencyServiceImpl.CRYPTO_FUNCTION;
+import static com.exonum.binding.cryptocurrency.transactions.CryptocurrencyTransactionTemplate.newCryptocurrencyTransactionBuilder;
+
+import com.exonum.binding.crypto.KeyPair;
+import com.exonum.binding.crypto.PublicKey;
+import com.exonum.binding.messages.BinaryMessage;
+import com.exonum.binding.messages.Message;
+import com.google.protobuf.ByteString;
+
+public class CreateWalletTxBase {
```

Comment:
The name is confusing — shan't it be `CreateWalletMessages` or `CreateWalletTransactionMessages`?

It's likely this class doesn't have to be extracted — see above.

Reply:
Could be renamed, i just want to extract common functionality to some helper classes. Don't like when it's borrowed in tests.